### PR TITLE
add ldexp torch frontend

### LINKED ISSUE #15517: ldexp

### PR DESCRIPTION
**Pull request by Lucas-Strange**
Close #15517

### CODE REVIEW

**Comment by Lucas-Strange** [comment]
Hi @theRealBird , please could you review my PR. Thank you

**Comment by theRealBird** [comment]
Hey @Lucas-Strange, 
I will definitely have a look. Thank you for your patience.😃

File: ivy/functional/frontends/torch/pointwise_ops.py
Comment (theRealBird) [inline ivy/functional/frontends/torch/pointwise_ops.py:1]: Hey @Lucas-Strange, could you please explain the reason behind not offering support for ```float16``` Thanks!

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**